v2/websocket/transport.go

```diff
@@ -85,8 +90,9 @@ func (w *ws) Send(ctx context.Context, msg interface{}) error {
 		return fmt.Errorf("websocket connection closed")
 	default:
 	}
-	w.log.Debug("ws->srv: %s", string(bs))
-	err = w.ws.WriteMessage(websocket.TextMessage, bs)
+	w.log.Info("ws->srv: %s", string(bs))
```

Comment:
does it make sense to keep it under debug level?

Reply:
Yep good point, changing back to debug